NOTICE

```diff
@@ -1,5 +1,5 @@
 Apache log4j
-Copyright 2010 The Apache Software Foundation
+Copyright 2021 The Apache Software Foundation
```

Comment:
2010-2021?

Reply:
Package was published on alphaWorks in 1999, moved to apache in 2000
https://web.archive.org/web/19991128202355/http://www.alphaworks.ibm.com/tech/logpackage
Initial apache svn commit https://svn.apache.org/viewvc/logging/log4j/trunk/build.xml?view=log&pathrev=308873 says 2000.
Matches with https://logging.apache.org/log4j/1.2/changes-report.html

https://www.apache.org/legal/src-headers.html#notice indeed says to use past-current.
So then it should be 2000-2021 :-)
Will change.